recorder/src/main/cpp/controller.cc

```diff
@@ -246,7 +250,7 @@ void Controller::run() {
     curl_easy_setopt(curl.get(), CURLOPT_HTTPHEADER, header_list.get());
     std::string service_endpoint_url = cfg.service_endpoint + std::string("/association");
     curl_easy_setopt(curl.get(), CURLOPT_URL, service_endpoint_url.c_str());
-    curl_easy_setopt(curl.get(), CURLOPT_UPLOAD, 1L);
+    curl_easy_setopt(curl.get(), CURLOPT_POST, 1L);
```

Comment:
We discussed PUT to POST only for /poll and /profile API
/association is implemented as PUT in backend right now